go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/crypto-power/cryptopower
 
-go 1.19
+go 1.20
 
 require (
-	decred.org/dcrwallet/v3 v3.0.1
+	decred.org/dcrwallet/v4 v4.0.0-20230809150859-a87fa843495e
```

Comment:
i don't see this tag in the dcrwallet repo

Reply:
You are right, Its not yet tagged but there in that version lies the solutions to the tickets mess we have been having.

Reply:
@dmigwi looking at this again, wouldn't it be ideal to leave the v3 and use the replace directive to point to v4

Reply:
Right. Was going to comment when I saw the PR title that there's no official dcrwallet v4 yet. The PR title should probably be upgrade to dcrwallet master. When dcrwallet v4 is tagged, we'd need to then do an upgrade to dcrwallet v4.